### DISCUSSION TITLE
Is Plyr using any Cookies?

### DISCUSSION THREAD

**Discussion by fiotakis** [sampotts/plyr#2420] [General]
Hello!

I would like to ask if Plyr uses any cookies.
I noticed that if the user change the speed of the video playing, then the next time that he will try to play again the video, the speed is stored.

I am asking this question because I want to use Plyr and Iwould like also to be compatible with GDPR issues.

Thank you in advance!
George

**Comment by fekle** [comment]
As far as I can tell, settings like the playback speed are stored in `localStorage`, not cookies. See the `storage` option in the README. 
If you play videos from external sources such as youtube, those may drop cookies, but that's not really Plyr's problem.

**Comment by Hashen110** [comment]
Plyr does not use cookies
